turkce/catalla-kaz/kus/kus.md

```diff
@@ -0,0 +1,13 @@
+Kusmaya başladın.
+
+O ses kulaklarından hiç gitmediği için sürekli kusmaya devam ediyordun.
+
+Biraz duvarları yesem geçer diye düşündün.
+
+Ama ya geçmezse?
+
+Bir yol bulup ilerlemek de bir seçenekti.
+
+Ne yapacaksın?
+
+[Duvarları yeme başla?](https://github.com/udacity/create-your-own-adventure/blob/master/turkce/duvarlari-ye/duvarlari-ye.md)
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.